src/Components/Cell/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Cell.css";
+const Cell = ({ cellId, value, handleClick }) => {
+  return (
+    <>
+      <div
+        className={`individual-cell ${
+          value === "X" ? "blue-color-text" : "red-color-text"
+        }`}
+        onClick={() => handleClick(cellId)}
+      >
+        {value}
```

Comment:
Instead of directly using the text to display `x` and `o`s, give them identifiable class names and try to use CSS to render the shapes. This might be an excellent opportunity to try and use CSS pseudo-elements. 

Kevin Powell's [tutorial](https://www.youtube.com/watch?v=zGiirUiWslI) on it is a great resource if you're unfamiliar with its usage.